unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/Editor/Procedural Skybox/SkyboxEditor_v2.cs

```diff
@@ -0,0 +1,1613 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace DCL.Skybox
+{
+    public enum SkyboxEditorToolsParts
+    {
+        Timeline_Tags,
+        BG_Layer,
+        Ambient_Layer,
+        Avatar_Layer,
+        Fog_Layer,
+        Directional_Light_Layer,
+        Base_Skybox
+    }
+
+    public class RightPanelPins
+    {
+        public string name;
+        public SkyboxEditorToolsParts part;
+        public int baseSkyboxSelectedIndex;
+        public bool pinned;
+        public Vector2 scroll;
+    }
+
+    public class SkyboxEditorWindow_v2 : EditorWindow
```

Comment:
Can we please add more file modularity here? A big file like this will have issues when managing conflicts and changes for multiple contributors. Also it would make testing and debugging more costly than it should. Maybe we can wrap some of the functionality around other classes and manage them from here.

Reply:
If there are functions that are not very reliant on this class state, we can wrap them up as static under a `SkyboxEditorUtils` static class. (Using that approach also makes implementing tests way easier down the road).

Reply:
Done. Divided this class in separate static classes which will handle their particular functionalities.